.prettierrc

```diff
@@ -1,9 +1,9 @@
 {
-  "semi": false,
```

Comment:
Not too opinionated on this, but I don't see a good reason to change it.

Reply:
boo